[2025-06-17 16:07:06,194] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2025-01-11T00:00:00+00:00 [queued]>
[2025-06-17 16:07:06,292] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2025-01-11T00:00:00+00:00 [queued]>
[2025-06-17 16:07:06,292] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2025-06-17 16:07:06,292] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2025-06-17 16:07:06,292] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2025-06-17 16:07:06,328] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2025-01-11T00:00:00+00:00
[2025-06-17 16:07:06,368] {standard_task_runner.py:52} INFO - Started process 15871 to run task
[2025-06-17 16:07:06,456] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'second_task', '2025-01-11T00:00:00+00:00', '--job-id', '7609', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpn9qzr_xv', '--error-file', '/tmp/tmp_xxv2ezj']
[2025-06-17 16:07:06,521] {standard_task_runner.py:77} INFO - Job 7609: Subtask second_task
[2025-06-17 16:07:07,188] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.second_task 2025-01-11T00:00:00+00:00 [running]> on host 4f8b76136637
[2025-06-17 16:07:07,608] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2025-01-11T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-01-11T00:00:00+00:00
[2025-06-17 16:07:07,613] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2025-06-17 16:07:07,616] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2025-06-17 16:07:07,662] {bash.py:169} INFO - Output:
[2025-06-17 16:07:07,666] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2025-06-17 16:07:07,667] {bash.py:177} INFO - Command exited with return code 0
[2025-06-17 16:07:07,878] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20250111T000000, start_date=20250617T160706, end_date=20250617T160707
[2025-06-17 16:07:08,062] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-17 16:07:08,109] {local_task_job.py:146} INFO - Task exited with return code 0
